modules/admin/proc-migrating-databases-to-an-external-server.adoc

```diff
@@ -0,0 +1,95 @@
+[id="proc-migrating-databases-to-an-external-server_{context}"]
+= Migrating local databases to an external database server
+
+By default, {product} hosts the data for each plugin in a PostgreSQL database. When you fetch the list of databases, you might see multiple databases based on the number of plugins configured. You can migrate the data from an {product-very-short} instance hosted on a local PostgreSQL server to an external PostgreSQL service, such as AWS RDS or Azure database. To migrate the data from each {product-very-short} instance one by one, you can use PostgreSQL utilities, such as link:https://www.postgresql.org/docs/current/app-pgdump.html[pg_dump] with link:https://www.postgresql.org/docs/current/app-psql.html[psql] or link:https://www.pgadmin.org/[pgAdmin]. 
+
+[NOTE]
+====
+For quick migration, you can also use this link:https://github.com/janus-idp/operator/blob/main/hack/db_copy.sh[database copy script].
+====
+
+.Prerequisites
+
+* You have installed the link:https://www.postgresql.org/docs/current/app-pgdump.html[pg_dump] and link:https://www.postgresql.org/docs/current/app-psql.html[psql] utilities on your local machine.
+* For data export, you have the PGSQL user sufficient privileges to make a full dump of local databases.
+* For data import, you have the PGSQL user sufficient admin privileges to create an external database and populate it with database dumps.
+
+.Procedure
+
+. Configure port forwarding for the local PostgreSQL database pod by running the following command on a dedicated terminal: 
++
+[source,terminal]
+----
+kubectl port-forward -n <your-namespace> <pgsql-pod-name> <forward-to-port>:<forward-from-port>
+----
+Where:
+* The `<pgsql-pod-name>` variable denotes the name of a PostgreSQL pod with the format `backstage-psql--<_index>`.
+* The `<forward-to-port>` variable denotes the port of your choice to forward PostgreSQL data to.
+* The `<forward-from-port>` variable denotes the local PostgreSQL instance port, usually `5432`.
++
+.Example: Configuring port forwarding
+[source,terminal]
+----
+kubectl port-forward -n backstage backstage-psql-backstage1-0 15432:5432
+Forwarding from 127.0.0.1:15432 -> 5432
+Forwarding from [::1]:15432 -> 5432
+----
```

Comment:
Can we use `oc` cmd instead of `kubectl` for this procedure? we should not document things for generic k8s, instead we need to make it specific for OCP.